include/picongpu/param/incidentField.param

```diff
@@ -19,7 +19,7 @@
 
 /** @file incidentField.param
  *
- * Configure incident field profile and gap between Huygence surface and absorber for each boundary.

```

Comment:
This was either forgotten or missed during rebase when changing gap to offset a couple weeks ago.